documentation/style-guide.rst

```diff
@@ -79,6 +84,11 @@ reST
    used to produce Python documentation.  When spelled out, it is
    always one word and both forms start with a lowercase 'r'.
 
+time zone
+   Spell out as two words when talking about the real-world concept.
+   Only spell as one word when referring to a Python term like a module,
+   class, or argument.
```

Comment:
```suggestion
   When referring to a Python term like a module, class, or argument
   it is always spelled as one word and surrounded by the appropriate markup
   (for example, ``:mod:`timezone```).
   When talking about the real-world concept it is spelled as two words.
```

Reply:
I added a modified version of this text.